src/signals-unix.c

```diff
@@ -350,6 +358,13 @@ int is_write_fault(void *context) {
     ucontext_t *ctx = (ucontext_t*)context;
     return exc_reg_is_write_fault(ctx->uc_mcontext.mc_err);
 }
+#elif defined(_OS_FREEBSD_) && defined(_CPU_AARCH64_)
+// FreeBSD seems not to expose a means of accessing ESR via `ucontext_t` on AArch64.
+// TODO: Is there an alternative approach that can be taken? ESR may become accessible
```

Comment:
@vtjnash, this seems like something you'd know: at least in theory, is it possible to implement `is_write_fault` on AArch64 given a `ucontext_t` that can't directly access ESR?